File: CO-SCHOOL-ANGULAR/src/app/Models/ResultDto.ts
import { Semester } from "./enums/enum";

export interface ResultDto {
    username: string;
    matter: string;
    semester: Semester;
    classRoomName: string;
    school_name:string;
    firstNote?: number;   
    secondNote?: number;  
    thirdNote?: number;   
    fourthNote?: number;  
    activitiesNote?: number; 
}

File: CO-SCHOOL-SPRING/src/main/java/com/coschool/dto/ResultDto.java
package com.coschool.dto;

import com.coschool.enums.Semester;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class ResultDto {
        private String username;
        private String matter;
        private Semester semester;
        private String classRoomName;
        private String school_name;
        private Double firstNote;
        private Double secondNote;
        private Double thirdNote;
        private Double fourthNote;
        private Double activitiesNote;
}
